rpi-update

```diff
@@ -296,13 +296,24 @@ function noobs_fix {
 
 }
 
+echo " *** Raspberry Pi firmware updater by Hexxeh, enhanced by AndrewS and Dom"
+echo
+echo " *** This script cannot uninstall any firmware automatically, only update it to a newer one. Use with caution."
```

Comment:
"only update it to a newer one" isn't strictly true - you can "update" to an older version, which is very useful for finding when an error was introduced. I'm happy for there to be a warning, but I'd like it to be correct.